packages/fuels-test-helpers/src/wallets_config.rs

```diff
@@ -19,16 +20,18 @@ impl WalletsConfig {
     ) -> Self {
         Self {
             num_wallets: num_wallets.unwrap_or(DEFAULT_NUM_WALLETS),
-            coins_per_wallet: coins_per_wallet.unwrap_or(DEFAULT_NUM_COINS),
-            coin_amount: coin_amount.unwrap_or(DEFAULT_COIN_AMOUNT),
+            assets: vec![(
+                AssetId::default(),
+                coins_per_wallet.unwrap_or(DEFAULT_NUM_COINS),
+                coin_amount.unwrap_or(DEFAULT_COIN_AMOUNT),
+            )],
         }
     }
 
-    pub fn new_single(coins: Option<u64>, amount: Option<u64>) -> Self {
+    pub fn new_multiple_assets(num_wallets: u64, assets: Vec<(AssetId, u64, u64)>) -> Self {
         Self {
-            num_wallets: 1,
-            coins_per_wallet: coins.unwrap_or(DEFAULT_NUM_COINS),
-            coin_amount: amount.unwrap_or(DEFAULT_COIN_AMOUNT),
```

Comment:
I don't think this setup function should disappear, it's quite practical to be able to just generate a new single wallet without having to specify assets.

Reply:
It is basically the same as `new` but the first parameter is set to 1 and it was used once...

Reply:
Ok I thought it was more used than that. I just always want make sure that we keep very simple and easy defaults available.